docs/source/getting_started.rst

```diff
@@ -0,0 +1,21 @@
+Getting Started
+===============
+
+The Data Resource Generator (DRG) is a database and API as configuration application.
+
+Terms
+-----
+
+Data Resource
+    These are things that can hold and handle data. In this case it is a database with a RESTful API
+
+Data Resource Schema
+    This is a database and API definition file.
+
+Checklist To Start
```

Comment:
I find this subsection a little confusing, since "Checklist" generally signals a list of things you need, like, "I need eggs, flour, and milk to make pancakes." I suggest calling this section "Steps to Get Started" and add a prefatory sentence:

> Steps to Get Started
> You can easily standup a DRG in four easy steps! 
> #. Create a Data Resource Schema [ link to instructions for doing this ]
> #. Run the Data Resource Generator [ link to instructions for doing this ]
> #. Generate your data resources [ link to instructions for doing this ]
> #. Profit!